### PR TITLE
Fixes the Ark's sound cutting out

### PR DESCRIPTION
**Pull request by ChangelingRain**
This is a thing now.

### CODE REVIEW

File: code/__DEFINES/sound.dm [hunk 1]
```diff
@@ -3,9 +3,10 @@
 #define CHANNEL_ADMIN 1023
 #define CHANNEL_VOX 1022
 #define CHANNEL_JUKEBOX 1021
+#define CHANNEL_JUSTICAR_ARK 1020
```
Comment (Xhuis) [inline code/__DEFINES/sound.dm]: It's "Justiciar", right?

**Comment by optimumtact** [comment]
So you're seizing an entire sound channel for a sound that appears only in a single gamemode and only at the end?

Seems snowflake

**Comment by lzimann** [comment]
Just pick the highest channel available instead.

**Comment by ChangelingRain** [comment]
The issue is we cycle sound channels too quickly for what is effectively a 5-minute long sound. Putting it on the highest channel won't fix the bug.

**Review by Cyberboss — APPROVED** [review]

**Comment by optimumtact** [comment]
what if it only used the channel during clockcult rounds, or add a system to reserve a lock on a channel and use that during ratvar spawning

**Comment by Cyberboss** [comment]
We can talk about that when we have 500 reserved channels

**Pull request merged**